modules/op-resolver-tasks-step-actions-about.adoc

```diff
@@ -0,0 +1,60 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: CONCEPT
+[id="resolver-tasks-step-actions-about_{context}"]
+
+= About non-versioned and versioned tasks and step actions
+
+The `openshift-pipelines` namespace includes versioned tasks and step actions alongside standard non-versioned tasks and step actions. For example, installing the {pipelines-title} Operator v1.16 creates `buildah-1-16-0` versioned task, `buildah` non-versioned task, `git-clone-1-16-0` versioned `StepAction` definition, and `git-clone` non-versioned `StepAction` definition.
```

Comment:
```suggestion
The `openshift-pipelines` namespace includes versioned tasks and step actions alongside standard non-versioned tasks and step actions. For example, installing the {pipelines-title} Operator v1.16 creates a `buildah-1-16-0` versioned task, a `buildah` non-versioned task, a `git-clone-1-16-0` versioned `StepAction` definition, and a `git-clone` non-versioned `StepAction` definition.
```
or maybe:
```suggestion
The `openshift-pipelines` namespace includes versioned tasks and step actions alongside standard non-versioned tasks and step actions. For example, installing the {pipelines-title} Operator v1.16 creates the following items:

 * `buildah-1-16-0` versioned task
 * `buildah` non-versioned task
 * `git-clone-1-16-0` versioned `StepAction` definition
 * `git-clone` non-versioned `StepAction` definition
```
("items", "resources", "artifacts", whichever you like / is accurate)

Reply:
I really like this, thank you!